docs/en_US/install/wiiu.md

```diff
@@ -55,17 +55,41 @@ Bleeding edge builds have not been widely tested, and do not guarantee any stabi
 
 <img src="/assets/images/docs/install/wiiu/aroma/inkay-nightly-link.png" width="100%">
 
+## Nimble
+Nimble is a Aroma plugin that allows your console to recive Spotpass data again. Download the latest version from [Nimble's latest release](https://github.com/PretendoNetwork/Nimble/releases/latest) on GitHub and download the `30_nimble.rpx` file from the latest release.
+
+<img src="/assets/images/docs/install/wiiu/aroma/nimbus-install.png" width="100%">
+
 ## Installation
 Place the downloaded `Inkay-pretendo.wps` file on your SD card at `sd:/wiiu/environments/aroma/plugins`. It's normal for other Aroma plugins to be in this folder too.
 
 <img src="/assets/images/docs/install/wiiu/aroma/sd-card.png" width="100%">
 
+Place the downloaded `30_nimble.rpx` file on your SD card at `sd:/wiiu/environments/aroma/modules/setup`. It's normal for other Aroma modules to be in this folder too.
+
 Place your SD card back into your console and boot like normal. You should see a notification of your connection:
 
 <img src="/assets/images/docs/install/wiiu/aroma/working-notification.jpg" width="100%">
 
 You are now connected to Pretendo Network.
 
+## After you set up your PNID on your Wii U.
+Shut off your console, and put the SD Card into your PC.
+
+Place the downloaded `30_nimble.rpx` file on your SD card at `sd:/wiiu/environments/aroma/modules/setup`. It's normal for other Aroma modules to be in this folder too.
+
+<img src="/assets/images/docs/install/wiiu/pnid/nimbus-sd.png" width="100%">
+
+Turn on your console
+
+Press `L + Down + SELECT` on the Wii U GamePad to open the Aroma plugin menu.
+
+<img src="/assets/images/docs/install/wiiu/pnid/aroma-plugins-list.jpg" width="100%">
+
+Select 'Inkay', then choose BOSS settings, then click 'Unregister Wara Wara Plaza BOSS tasks'
+
+Restart your console.
+
```

Comment:
You duplicated a lot of this information in various places, I feel like this could be structured differently?

Reply:
Thanks for the feedback, I'll try and fix some stuff up.